tensorflow_datasets/core/dataset_builder.py

```diff
@@ -166,6 +166,8 @@ class DatasetBuilder(object):
   # displayed in the dataset documentation.
   MANUAL_DOWNLOAD_INSTRUCTIONS = None
 
+  # This flag should be set to `True` if the dataset is defined outside tfds.
+  OUTSIDE_TFDS = False
```

Comment:
I don't think we should add a new flag.
TFDS should automatically detect whether the checksums,... are defined, then fallback to default values otherwise.